crates/cli/src/options.rs

```diff
@@ -9,4 +9,7 @@ pub struct Options {
 
     #[structopt(short = "o", parse(from_os_str), default_value = "index.wasm")]
     pub output: PathBuf,
+
+    #[structopt(short = "d")]
+    pub dynamic: bool,
```

Comment:
Can we [add a description ](https://docs.rs/structopt/0.2.6/structopt/#help-messages)here so that the message is available when running with `--help`?